source/2020-07-11-the-ember-times-issue-155.md

```diff
@@ -29,36 +32,42 @@ To help test beta builds and report any bugs before they are published as a fina
+There's an updated version of [Balint Erdi's (@balinterdi)](https://github.com/balinterdi) excellent book Rock & Roll with Ember.js available, now with coverage for Ember's Octane Edition! 🎉
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+If you have not had a chance to give Rock & Roll with Ember.js a read, you should. This is one of the most well-written books on Ember.js. It's a great way to introduce people new to Ember to the framework's conventions and teach them how to write idiomatic Ember code. 
+
+Using a pragmatic approach of learning by building, Rock & Roll with Ember.js takes the reader through the process of building an Ember application from the ground up. Readers should expect to finish the book with a solid understanding of the fundamentals involved in building Ember applications.
+
+You can find the book [here](https://www.balinterdi.com/rock-and-roll-with-emberjs/).  
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [State of Frontend Survey 📊](https://tsh.io/state-of-frontend/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Interested in helping Ember.js 🔥 get the recognition it deserves among other frontend tools? You may want to participate in the State of Frontend Survey.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<!--alex ignore easy of-course-->
+This survey aims to gather information on the reality of frontend developers' work. It gives us a chance to provide input on tooling we use (Ember of course!). 
+
+It also allows participants the opportunity to comment on tooling they would like to learn and provide thoughts about the future of frontend development trends.
+
+The survey should only take 7 minutes or so of your time and could help promote the visibility of Ember.js and work to ensure the survey provides accurate information about what tools developers are using. 
+
+Take a moment and fill out the survey [here](https://tsh.io/state-of-frontend/).
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [ember-meta 1.0.0 release 🚢](https://twitter.com/shipshapecode/status/1281406393577283587)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+`ember-meta 1.0.0` has been released by [Ship Shape (@shipshapecode)](https://github.com/shipshapecode)! Thanks to [Chris Manson (@mansona)](https://github.com/mansona) for his continued help with this addon 🎉!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In case you weren't aware `ember-meta` is an addon that helps setup meta for your Prember/Ember blog to support opengraph, microdata, Facebook, Twitter, Slack, etc.
```

Comment:
Wasn't sure if you listed Facebook separately from Open Graph for a reason. In case it was a duplicate, I removed it in the suggestion below.

```suggestion
In case you weren't aware, `ember-meta` is an addon that helps set up `<meta>` tags for your Prember/Ember blog. These tags can support Open Graph, microdata, Twitter, Slack, etc.
```